hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFileInterfacesV1.java

```diff
@@ -44,9 +43,7 @@ public TestOzoneFileInterfacesV1(boolean setDefaultFs,
   @Override
   protected OzoneConfiguration getOzoneConfiguration() {
     OzoneConfiguration conf = new OzoneConfiguration();
-    conf.setBoolean(OMConfigKeys.OZONE_OM_ENABLE_FILESYSTEM_PATHS,
-            enableFileSystemPaths);
-    conf.set(OMConfigKeys.OZONE_OM_LAYOUT_VERSION, "V1");
```

Comment:
Is here correct to just delete this rather than OzoneTestUtils.configureFSOptimizedPaths(conf, enableFileSystemPaths, OMConfigKeys.OZONE_OM_LAYOUT_VERSION_V1);?

Reply:
Good catch. I have completely missed adding new function.